### DISCUSSION TITLE
Alternate Noco Logo

### DISCUSSION THREAD

**Discussion by distresscode** [nocodb/nocodb#4249] [Self hosting]
I use Cloudron to host Noco and a plethora of other apps. I didn't like Noco's droid logo in my dashboard so I modified the "N" logo to make it more visible when dark mode is active. 

![noco](https://user-images.githubusercontent.com/2686608/199125026-5ee7fb44-0ae4-450b-8fdb-fe1277189f0d.png)
![preview](https://user-images.githubusercontent.com/2686608/199125059-3b459b51-d8ac-4ccd-bbbb-05dca4a877bb.png)

PSD File [noco.zip](https://github.com/nocodb/nocodb/files/9905507/noco.zip). _Please message me if the file isn't working as I used Affinity Photo to export the file and not Photoshop._

**Comment by o1lab** [comment]
Thank you & appreciate it. Could you not set the green icon to pinkish that our brand has. I will also provide this as input to our designer.

**Comment by distresscode** [comment]
![noco2](https://user-images.githubusercontent.com/2686608/199724954-e2e4e719-61af-42bd-9196-7e9903e80d2a.png)
